src/content/desci/index.md

```diff
@@ -8,77 +8,267 @@ sidebar: true
+Decentralized science (DeSci) is a movement that aims to build public infrastructure for funding, creating, reviewing, crediting, storing, and disseminating scientific knowledge fairly and equitably using the web3 stack.
+
+The goal of DeSci is to create an ecosystem in which scientists are incentivized to openly share their research and receive credit for their work, while also providing a way for the general public to easily access and contribute to scientific research. DeSci is built on the premise that scientific knowledge should be open and accessible to everyone, and that the process of scientific research should be transparent. DeSci is also intended to create a more decentralized and distributed model for scientific research, which would make it more resistant to censorship and control by central authorities. By decentralizing access to funding, scientific tools and communication channels, DeSci hopes to create an environment where new and unconventional ideas can flourish.
+
+## The problems in science
+
+Science has been failing to keep up with the demands of the modern world for centuries. The mechanisms that have governed scientific inquiry for centuries are no longer providing all the necessary advancements needed in medicine, energy production, materials, exploration, mathematics, and other areas of scientific development. 
+
+This is due in part to the gatekeeping of knowledge and resources by third party arbiters, as well as the inability of the general public to participate in and understand the scientific process. 
+
+Decentralized science can help to address these issues by allowing for more diverse funding sources (from DAOs, quadratic donations to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.
+
+​​
+
+
+## Why we need science
+
+Science is a process that allows us to create and translate knowledge in order to improve the quality of life for humanity as a whole. It also provides solutions to some of the most complex problems we face. In order to be trustworthy, knowledge produced by science must be replicable and open to scrutiny. This process should be impartial and available to the public. Any knowledge that cannot be replicated or tested for falsifiability should not be considered as the truth.
+
+By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.
+
+
+## How DeSci improves science
```

Comment:
Maybe create a "yes and" visualization. Traditional: National agency. Desci: National agency AND x,y,z